### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by alecchangod** [ROM-builders/temporary#21682] [Q&A]
Sir my account get suspended by cirrus. Can you add my new account into team please? Thanks.
https://cirrus-ci.com/build/4796377423675392

New account: @alecchan02

**Comment by Apon77** [comment]
a few days ago you asked for alecchan001, now for alecchan02

**Reply by alecchangod** [reply]
That's one month ago. Idk why that account get suspended too, so make another account
